### PR TITLE
Created a camel-pdf component extension fixes #341

### LINKED ISSUE #341: Create a Camel pdf component extension

### PR DESCRIPTION
**Pull request by aldettinger**

### CODE REVIEW

File: integration-tests/pdf/pom.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.camel.quarkus</groupId>
+    <artifactId>camel-quarkus-integration-tests</artifactId>
+    <version>0.3.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>camel-quarkus-integration-test-pdf</artifactId>
+  <name>Camel Quarkus :: Integration Tests :: Pdf</name>
```
Comment (ppalaga) [inline integration-tests/pdf/pom.xml]:
```suggestion
  <name>Camel Quarkus :: Integration Tests :: PDF</name>
```

File: integration-tests/pdf/pom.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  <description>Integration tests for Camel Quarkus Pdf extension</description>
```
Comment (ppalaga) [inline integration-tests/pdf/pom.xml]:
```suggestion
  <description>Integration tests for Camel Quarkus PDF extension</description>
```

File: extensions/pdf/runtime/src/main/resources/META-INF/quarkus-extension.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Camel Quarkus Pdf",
+  "labels": [
+    "integration",
+    "camel"
```
Comment (ppalaga) [inline extensions/pdf/runtime/src/main/resources/META-INF/quarkus-extension.json]:
```suggestion
    "camel",
    "PDF"
```

**Comment by ppalaga** [comment]
BTW, if the usage of this extension somehow differs from the stock Camel PDF component, it would be nice to add a docs page `docs/modules/ROOT/pages/extensions/pdf.adoc`.

**Comment by aldettinger** [comment]
Thanks for reviews and suggestions guys.

@ppalaga At first sight, I don't think that the usage is that different from the stock pdf component. Yet, I would like to align the doc to what is done in other extensions. Are you ok if we push this PR as this and I check the doc in a separate commit ?

**Comment by oscerd** [comment]
+1 for the docs in a separate PR. Lets wait for the CI build to finish.

**Comment by ppalaga** [comment]
> @ppalaga At first sight, I don't think that the usage is that different from the stock pdf component. Yet, I would like to align the doc to what is done in other extensions. Are you ok if we push this PR as this and I check the doc in a separate commit ?

No problem with a separate PR. But please note that the doc page is optional. Add it only if you have some meaningful info to put there.

**Comment by aldettinger** [comment]
Ok, I've reviewed the doc pages for other extensions and I don't think we need one here. So, I'm gonna merge when the pr build check is ok. Thanks guys :+1:

**Comment by oscerd** [comment]
Thanks

**Pull request merged**